cloudbuild.yaml

```diff
@@ -256,12 +257,16 @@ steps:
         python3 test_frontend.py "127.0.0.1:8081"
         echo "pass" > /workspace/rag_frontend_result.txt
 
-        # Upload locally stored netflix dataset to GCS bucket mounted as /data
-        gsutil cp ./netflix_titles.csv gs://gke-aieco-rag-$SHORT_SHA-$_BUILD_ID/netflix-shows/netflix_titles.csv
-        ray job submit --working-dir . --address=http://127.0.0.1:8265 -- python ray_job.py
-
-        python3 test_rag.py "http://127.0.0.1:8081/prompt"
-        echo "pass" > /workspace/rag_prompt_result.txt
+        cd /workspace/
+        sed -i "s/<username>/$$KAGGLE_USERNAME/g" ./applications/rag/example_notebooks/rag-kaggle-ray-sql-interactive.ipynb
+        sed -i "s/<token>/$$KAGGLE_KEY/g" ./applications/rag/example_notebooks/rag-kaggle-ray-sql-interactive.ipynb
+        gsutil cp ./applications/rag/example_notebooks/rag-kaggle-ray-sql-interactive.ipynb gs://gke-aieco-rag-$SHORT_SHA-$_BUILD_ID/
+        kubectl exec -it -n rag-$SHORT_SHA-$_BUILD_ID $(kubectl get pod -l app=jupyterhub,component=hub -n rag-$SHORT_SHA-$_BUILD_ID -o jsonpath="{.items[0].metadata.name}") -- jupyterhub token admin --log-level=CRITICAL | xargs python3 ./applications/rag/notebook_starter.py
```

Comment:
I think `./applications/rag/notebook_starter.py` can just run locally because we enabled port-forwrarding right?

Reply:
Ah nvm I see that it already is

Reply:
Oh, does it not get ran locally? I assumed that it does since I piped it and the pod doesn't have `notebook_starter.py`.

Reply:
Oh oops I didn't update the page so only saw your first comment